### DISCUSSION TITLE
Make an Installer that Gives Us an .Exe

### DISCUSSION THREAD

**Discussion by ToddStarbuck** [comfyanonymous/ComfyUI#116] [Ideas]
Here's how it should work:

1. Download the installer
2. Run the installer
3. Click next a few times
4. Icon appears on desktop
5. Double click the icon and the program runs

**Comment by WASasquatch** [comment]
Lots of steps over just extracting the file and running it. Python installers are rather gimmicky.

**Comment by AdrianTheFrog** [comment]
There are multiple programs that automatically convert a .bat file to a .exe file, but I can't guarantee that they don't also give malware.
